perf.groovy

```diff
@@ -639,13 +651,11 @@ parallel(
 
                     if (isPR) {
                         def opts = ""
-                        if (opt_level == 'min_opts')
-                        {
+                        if (opt_level == 'min_opts') {
```

Comment:
Overall, the change looks fine to me.  One thing that I would like to know: Is there a style guide for *.groovy files?  I notice that much of your change is moving '{' characters onto the end of the previous line.  Is there clear guidance for this?

Adding @mmitche to comment.